hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsConfiguration.kt

```diff
@@ -70,5 +71,8 @@ class HmppsSqsConfiguration {
   @Bean
   @ConditionalOnMissingBean
   @DependsOn("hmppsQueueService")
```

Comment:
I think this should also depend on the hmppsAsyncQueueService (missed from my last PR).